Documentation/ApiOverview/Fal/Administration/Storages.rst

```diff
@@ -53,3 +53,33 @@ Is online?
         within your web root or accessible via a third-party storage service
         which is publicly available. The files will still be available to anyone
         who knows the path to the file.
+
+    ..  versionchanged:: 11.5.35/12.4.11
+
+    Assuming that a web project is located in the directory
+    :file:`/var/www/example.org/` (the "project root path" for Composer-based
+    projects) and the publicly accessible directory is located at
+    :file:`/var/www/example.org/public/` (the "public root path"), accessing
+    resources via the File Abstraction Layer component is limited to the
+    mentioned directories.
```

Comment:
```suggestion
    mentioned directories and its sub-directories.
```

(I wonder though, why not ONLY mention the "project root path", because the "document root" is always a subdirectory of that, isn't it?)

Reply:
Have to look into the code of the patchset. As above, took it from the changelog, I assume to make it more clear. fileadmin is in web root, whereas a "private" storage is in the project root (have this, for example, for application uploads).